sympy/__init__.py

```diff
@@ -496,3 +496,171 @@ def __sympy_debug():
     # sympy.deprecated:
     'C', 'ClassRegistry', 'class_registry',
 ]
+
+
+#=======================================================================
+#
+# XXX: The names below were importable before sympy 1.6 using
+#
+#          from sympy import *
+#
+# This happened implicitly because there was no __all__ defined in this
+# __init__.py file. Not every package is imported. The list matches what
+# would have been imported before.
+#
+#=======================================================================
+
+
+__all__.extend([
+    'algebras',
+    'assumptions',
+    'calculus',
+    'codegen',
+    'combinatorics',
+    'concrete',
+    'deprecated',
+    'discrete',
+    'external',
+    'functions',
+    'geometry',
+    'interactive',
+    'multipledispatch',
+    'ntheory',
+    'parsing',
+    # 'physics',
+    'plotting',
+    'polys',
+    'printing',
+    'release',
+    'strategies',
+    'tensor',
+    'utilities',
+])
+
+
+#=======================================================================
+#
+# XXX: The names below were importable before sympy 1.6 using
+#
+#          from sympy import *
+#
+# This happened implicitly because there was no __all__ defined in this
+# __init__.py file. The plan is to remove them but for now they remain
+# importable but will give a deprecation warning when used.
+#
+#=======================================================================
```

Comment:
I don't understand the purpose of have this comment block repeated. It refers to the names below, but there is nothing below?

Reply:
There's a bug list of names in `_DEPRECATED_IMPORTS` below. The difference between these and those that are listed above is that these are deprecated and will give a deprecation warning. The ones above (in this PR currently) are not deprecated and will give no warning when used.